pylearn2/training_algorithms/tests/test_learning_rule.py

```diff
@@ -170,23 +170,34 @@ def adadelta_manual(model, state):
                izip(manual, model.get_params()))
 
 
-def test_adagrad():
+def prepare_adagrad_test(dataset_type='arrange', model_type='random'):
```

Comment:
Maybe "arange" instead of "arrange", it is named after `numpy.arange` (for "array" and "range"), not after the English word [arrange](http://en.wiktionary.org/wiki/arrange).